File: Python/URI/Beginner/1074.py
n = input()
n = int(n)

for i in range(n):
	a = input()
	a = int(a)

	if (a < 0 and a%2==0):
		print('EVEN NEGATIVE')
	elif (a < 0 and a%2==1):
		print('ODD NEGATIVE')
	elif (a > 0 and a%2==0):
		print('EVEN POSITIVE')
	elif (a > 0 and a%2==1):
		print('ODD POSITIVE')
	else:
		print('NULL')

File: URI/Beginner/1074.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {

    int n, x;
    scanf("%d", &x);

    while (x--){
        scanf("%d",&n);

        if (n == 0){
            printf("NULL\n");
        } else if (n%2==0){
            if (n > 0){
                printf("EVEN POSITIVE\n");
            } else {
                printf("EVEN NEGATIVE\n");
            }            
        } else if (n%2!=0) {
            if (n > 0) {
                printf("ODD POSITIVE\n");
            } else {
                printf("ODD NEGATIVE\n");
            }
        }
    }
    
    return 0;
}